src/shared/install.c

```diff
@@ -1912,6 +1912,16 @@ static int install_info_symlink_alias(
                 if (!alias_path)
                         return -ENOMEM;
 
+                if (!force) {
+                        q = chase_symlinks(alias_path, lp->root_dir, CHASE_NONEXISTENT, NULL, NULL);
+                        if (q < 0 && q != -ENOENT) {
+                                r = r < 0 ? r : q;
+                                continue;
+                        }
+                        if (q == 0)
+                                force = true;
```

Comment:
so if the file doesn't exist then you enable force mode? i don#t grok that logic? care to explain?